api/openapi/v2/system-agent.yaml

```diff
@@ -0,0 +1,634 @@
+openapi: 3.0.0
+info:
+  title: Edgex Foundry - System Management Agent API
+  description: This is the definition of the API for the System Management Agent service in the EdgeX Foundry IOT microservice platform. System Management Agent is responsible for management of EdgeX microservices and supported infrastructure.
+  version: "2.0"
+
+servers:
+  - url: "http://localhost:48090/api/v2"
+    description: URL for local development and testing
+
+components:
+  schemas:
+    BaseRequest:
+      description: "Defines basic properties which all use-case specific request DTO instances should support."
+      type: object
+      properties:
+        requestId:
+          description: "Uniquely identifies this request. For implementation, recommend this value be generated by the type's constructor."
+          type: string
+          format: uuid
+          example: "e6e8a2f4-eb14-4649-9e2b-175247911369"
+      required:
+        - requestId
+    BaseResponse:
+      description: "Defines basic properties which all use-case specific response DTO instances should support"
+      type: object
+      properties:
+        requestId:
+          description: "Uniquely identifies the request that resulted in this response."
+          type: string
+          format: uuid
+          example: "e6e8a2f4-eb14-4649-9e2b-175247911369"
+        statusCode:
+          description: "A numeric code signifying the operational status of the response."
+          type: integer
+        message:
+          description: "A field that can contain a free-form message, such as an error message."
+          type: string
+      required:
+        - requestId
+        - statusCode
+    ErrorResponse:
+      allOf:
+        - $ref: '#/components/schemas/BaseResponse'
+      description: "A response type for returning a generic error to the caller."
+      type: object
+    GetConfigResponse:
+      allOf:
+        - $ref: '#/components/schemas/BaseResponse'
+      description: "Provides a response containing the configuration for the targeted service."
+      type: object
+      properties:
+        config:
+          description: "A string-ified representation of the service's configuration. For purposes of this specification, a string has been used since configuration structure differs from service to service."
+          type: string
+      required:
+        - config
```

Comment:
![image](https://user-images.githubusercontent.com/464327/74246304-ec8bf280-4ca9-11ea-9c20-6189b02f5fab.png)

Note structure, not string.